packages/svelte/tests/validator/samples/window-binding-invalid-dimensions/errors.json

```diff
@@ -0,0 +1,14 @@
+[
+	{
+		"code": "bind_invalid_name",
+		"message": "`bind:clientWidth` is not a valid binding. Possible bindings for <svelte:window> are focused, clientWidth, clientHeight, offsetWidth, offsetHeight, contentRect, contentBoxSize, borderBoxSize, devicePixelContentBoxSize, this, innerText, innerHTML, textContent",
```

Comment:
that list of suggestions is not correct, which hints at that we need to add more bindings to the invalid ones. `this`, `innerText`, `innerHTML`, `textContent` don't work. The `contentRect` etc dimension bindings might (?) work, I'm not sure.

Reply:
Ugh i tried with all the dimensions but not with the rest...let me check the rest

Reply:
Uh also i have to check for `parent.name` not `node.name`...btw `this` works

Reply:
Mhm yeah I guess `this` would work. I just did a quick check in the Svelte 4 REPL and the number of bindings are restricted there. But if the other ones work I don't see why we wouldn't allow them.

Reply:
I've now tested all the bindings that the error was proposing me and all of them work.

Reply:
@dummdidumm just to be sure: is the last update ok or I need to something else?